server/protocols/jmap-draft/pom.xml

```diff
@@ -241,7 +241,7 @@
         <dependency>
             <groupId>org.jgrapht</groupId>
             <artifactId>jgrapht-core</artifactId>
-            <version>1.5.0</version>
+            <version>1.5.1</version>
```

Comment:
The commit is ` [UPGRADE] simpleclient_dropwizard 0.10.0 -> 0.11.0 `

This change is unrelated

Reply:
Mistake ;-)